helm-prereqs/cleanup-machine-a-tron.sh

```
@@ -0,0 +1,296 @@
+    # fresh mock boots at the factory password, but a surviving per-MAC entry
+    # tells site-explorer the BMC was already rotated, so it presents the old
+    # rotated password, gets 401 Unauthorized, and permanently latches
+    # AvoidLockout (NICO-SITEEXPLORER-144) on the endpoint.
+    # Batch server-side in ONE kubectl exec: at scale there are thousands of
+    # per-MAC entries and one exec per deletion takes HOURS (one round-trip +
+    # exec setup each); a shell loop on the vault pod does them all in seconds.
+    _n="$(vault_cmd 'count=0
+for m in $(vault kv list -format=yaml secrets/machines/bmc 2>/dev/null | sed "s/^- //" | grep -v "^site/"); do
+  vault kv metadata delete "secrets/machines/bmc/${m%/}/root" >/dev/null 2>&1 && count=$((count+1))
+done
+echo $count' || echo 0)"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)